packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -280,23 +280,39 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
       return;
     }
     const { volumeActor } = applicableVolumeActorInfo;
+    const vtkImageData = volumeActor.getMapper().getInputData();
+    const interval = vtkImageData.getSpacing()[2];
```

Comment:
it is always 1 so we still at the 0.9 corresponding to 0.5 opacity

Reply:
I don't understand, sorry. 
the spacing in [2] should not be really matter since based on the view (axial, sagittal, cor) the [2] might be in plane or out of plane.